data/human/intro missions.txt

```diff
@@ -508,3 +508,95 @@ conversation "end of intro missions"
 		
 	label end
 	`	He hands you a credit chip worth <payment>, and says, "Best of luck to you, Captain. Don't hesitate to look me up if you're ever in the area. And remember to check the spaceports wherever you land. You never know who you'll find wandering around there."`
+
+
+
+mission "Intro [Mining]"
+	landing
+	name "Explore Mining"
+	description "Learn about mining."
+	source "Sunracer"
```

Comment:
```suggestion
	invisible
```
You're just going to decline it at some point, it might as well be invisible. (Invisible might not go here though, I don't really know the order mission stuff usually goes.)